src/sonic-config-engine/sonic-cfggen

```diff
@@ -34,6 +34,18 @@ def is_ipv6(value):
             return False
     return addr.version == 6
 
+def get_machine_info():
+    if not os.path.isfile('/host/machine.conf'):
```

Comment:
I don't really want to limit the usage of this tool to only environment where /host/machine.conf exists, as most of the functionality actually don't depend on that.

---
In reply to: [101696547](https://github.com/Azure/sonic-buildimage/pull/304#discussion_r101696547) [](ancestors = 101696547)